_posts/2025-06-01-marcado-facturas.md

```
@@ -0,0 +1,27 @@
+
+Ahora puedes mantener tu portafolio actualizado al marcar tus facturas diariamente. Podrás asignarles distintos estados a tus facturas; estos son:
+- Riesgo
+- Tasa
+- Potencial
+- No califica
+- Ganada
+- Perdida
+
+Cuando asignes un estado a tus facturas, estas se procesarán automáticamente y formarán parte de tus indicadores en el `Reporte de gestión comercial`; solo podrás seguir visualizando aquellas que figuren como `Potencial`. Esas quedarán en la tabla para futuras modificaciones.
+
+<video width="100%" controls>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: video

(MD033, no-inline-html)